.semaphore/semaphore.yml

```diff
@@ -26,13 +27,11 @@ blocks:
           value: /Users/semaphore/git/flowcrypt-ios
       prologue:
         commands:
-          - checkout && cd ~/git/flowcrypt-ios/
-          - cache restore && make dependencies && cache store
+          - checkout && cd ~/git/flowcrypt-ios/ && cache restore && make dependencies && cache store
           - mv ~/appium-env ~/git/flowcrypt-ios/appium/.env
           - sem-version node 16 && cache restore appium-npm && cd ./appium && npm i && cd .. && cache store appium-npm appium/node_modules
           - brew install ideviceinstaller
-          - npm install ios-deploy -g --unsafe-perm=true --allow-root
-          - npm install ios-sim -g --unsafe-perm=true --allow-root
+          - npm install ios-deploy -g --unsafe-perm=true --allow-root && npm install ios-sim -g --unsafe-perm=true --allow-root
```

Comment:
By the way, do we use `ideviceinstaller`, `ios-deploy` and `ios-sim` libs?
As I haven't found any other mentions of them in the code

Reply:
I have no clue. Guess I could try removing them.

Reply:
I especially don't like they are not versioned. Should be installed using npm instead, if needed.